### PR TITLE
docs: add `check` to atomic op in `kv` example

### PR DESCRIPTION
**Pull request by adit-bala**

### CODE REVIEW

**Comment by adit-bala** [comment]
Hey @thisisjofrank! Could I ask for some feedback on this pr :D

File: examples/kv.ts [hunk 1]
```diff
@@ -73,12 +73,19 @@ const aliceScoreKey = ["scores", "alice"];
 await kv.set(aliceScoreKey, new Deno.KvU64(0n));
 
 // Add 10 to the player's score in an atomic transaction
-await kv.atomic()
+const res = await kv.atomic()
+  .check({ key: aliceScoreKey, versionstamp: null }) // Check if this key has been modified since we read it
```
[the remark this answers is not in the document]
Comment (adit-bala) [reply examples/kv.ts]: Yea, sorry I forgot actually to use the versionstamp. i don't know if this is the best place to show the functionality of an `AtomicCheck`, but I do think it is an important and powerful feature to display, do you think this has a place in this example, or should it be a separate one altogether?

**Comment by adit-bala** [comment]
@kt3k It should now work, I forgot actually to use the `versionStamp`. I don't know if this is the best place to show the functionality of an AtomicCheck, but I do think it is an important and powerful feature to display, do you think this has a place in this example, or should it be a separate one altogether?

**Review by kt3k — APPROVED** [review]
Looks great now! Thanks for updating!

**Pull request merged**